prow/config/config.yaml

```diff
@@ -129,6 +129,63 @@ branch-protection:
                   - "unit-test"
                   - "license/cla"
                 strict: true
+        tidb:
+          branches:
+            master:
+              enforce_admins: false
+              protect: true
+              required_status_checks:
+                contexts:
+                  - "idc-jenkins-ci-tidb/build"
+                  - "idc-jenkins-ci-tidb/check_dev"
+                  - "idc-jenkins-ci-tidb/check_dev_2"
+                  - "idc-jenkins-ci-tidb/check_release_note"
+                  - "license/cla"
+                strict: true
+            release-2.1:
+              enforce_admins: false
+              protect: true
+              required_status_checks:
+                contexts:
+                  - "idc-jenkins-ci-tidb/build"
+                  - "idc-jenkins-ci-tidb/check_dev"
+                  - "idc-jenkins-ci-tidb/check_dev_2"
+                  - "idc-jenkins-ci-tidb/check_release_note"
+                  - "license/cla"
+                strict: true
+            release-3.0:
```

Comment:
miss release 3.1

Reply:
There doesn't seem to be a version 3.1 branch on tidb.

<img width="405" alt="image" src="https://user-images.githubusercontent.com/29879298/108668043-583ba900-7515-11eb-856b-d2a82b2966ff.png">